components/tag-list/tag-list-item-mixin.js

```diff
@@ -63,12 +64,12 @@ export const TagListItemMixin = superclass => class extends LocalizeCoreElement(
 				overflow: hidden;
 				text-overflow: ellipsis;
 			}
-			:host(:focus) .tag-list-item-container,
-			:host(:focus:hover) .tag-list-item-container {
+			:host(:focus-visible) .tag-list-item-container,
```

Comment:
Hmmm, this might require us to use `forceFocusVisible` when moving focus from the "interactive" toggle.

Reply:
Ah - does that mean we shouldn't go with `focus-visible` here? Or is that just something that will need to be kept in mind to be tested?

Reply:
I think we should test this out in conjunction with my "interactive" changes.  I add a demo to the tag-list in my PR for that case.

Reply:
Sounds good, I'm good to wait until yours merges

Reply:
After merging your changes in this appears to still work as expected both in non-interactive and interactive mode